### DISCUSSION TITLE
sbctl and my hardware...

### DISCUSSION THREAD

**Discussion by DanteIgashu** [Foxboron/sbctl#324] [Q&A] [answered]
Hello!
First of all, THANKS a lot for such an useful tool, it made it really easy to activate "Secure Boot" on my laptop using EndeavourOS.

Now, my question is related to the next issue.
I tried to use "sbctl" on my desktop to activate "Secure Boot" on another EndeavourOS system, and I got this message:

          2024/07/20 23:39:06 Not implemented signature list certificate: 

Does it mean that I can't use this tool with my hardware? or I still can use it, but I may not get confirmation that secure boot is active while using "sbctl status"? Or what does that mean?

Useful (probably) Hardware info:
Motherboard: GA-AX370-Gaming K7 (Gigabyte)
Processor: AMD Ryzen 7 1800X

Thanks in advance for any help related to my problem! :D

**Comment by Foxboron** [comment]
Please dump the following files somewhere.

- `/sys/firmware/efi/efivars/db-d719b2cb-3d3a-4596-a3bc-dad00e67656f`
- `/sys/firmware/efi/efivars/KEK-8be4df61-93ca-11d2-aa0d-00e098032b8c`
- `/sys/firmware/efi/efivars/PK-8be4df61-93ca-11d2-aa0d-00e098032b8c`

Please always attach the complete output and which commands you are using. I'm unable to guess this.

**Comment by DanteIgashu** [comment]
Hello!

Yeah, I used "sbctl status" and that's the full output I got:

      2024/07/20 23:39:06 Not implemented signature list certificate: 

Nothing more...

About the files, I haven't created any keys yet, so none of those files exist in my system, and I haven't created the keys yet because I got the output that I mentioned first, and after that I decided to ask what I asked on my first post, so I wouldn't do something I could regret doing later, so I stopped at the first step.
The thing is that, when I activated secure boot on my laptop, I used "sbctl status" first and got the usual message that says if sbctl is installed, if setup mode in enabled, if secure boot is enabled and the key manufacturer or type (or something like that), but on my desktop that doesn't appear at all, I just get that message, and that's why I am asking if the message means I can't or shouldn't use sbctl with my system.

5 minutes later:
Okay, so i created the keys, but I still get the exact same message when I use "sbctl status". Do you still want me to get you the files you asked me for?

10 minutes later:
Okay, I checked if the files existed, but they don't, the system just says that, that the files don't exist.

15 minutes later:
I found the files, but they are named slightly differently, but, how do I dump them, my system doesn't have the "dump" software installed and what I find on the repositories is not called just "dump", they are called "dump_" and something else. Help please...

**Reply by Foxboron** [reply]
Upload the following files somewhere so I can download them.

- `/sys/firmware/efi/efivars/db-d719b2cb-3d3a-4596-a3bc-dad00e67656f`
- `/sys/firmware/efi/efivars/KEK-8be4df61-93ca-11d2-aa0d-00e098032b8c`
- `/sys/firmware/efi/efivars/PK-8be4df61-93ca-11d2-aa0d-00e098032b8c`
- `/sys/firmware/efi/efivars/dbDefault-8be4df61-93ca-11d2-aa0d-00e098032b8c`
- `/sys/firmware/efi/efivars/KEKDefault-8be4df61-93ca-11d2-aa0d-00e098032b8c`
- `/sys/firmware/efi/efivars/PKDefault-8be4df61-93ca-11d2-aa0d-00e098032b8c`

**Reply by DanteIgashu** [reply]
Update:
After upgrading my system, I noticed that the kernel images got signed, even thought I hadn't finished the secure boot activation process, so, after seeing that, I decided to complete the process, and now secure boot is active on my motherboard, Linux boots, but when I use "sbctl status", I still get the same message.

Files: https://ufile.io/de2f0p84
Password: a]cGz$QgH,xOXk,@^}%5?Nm]
I don't have a "/sys/firmware/efi/efivars/PKDefault-8be4df61-93ca-11d2-aa0d-00e098032b8c" file on my system. Does that indicate there's something wrong?.

Note: You could have told me that the files would be created once I finished the process to activate Secure Boot, but you didn't, if you knew that from the beginning, that was mean. They are there now that I activated secure boot, before there were only some files with the same names but they had an extra "x" in them, so I guess they are not the same files.

I hope the files I got for you serve their purpose! :D

**Comment by Foxboron** [comment, marked as answer]
Is this still an issue with the latest release?

**Reply by DanteIgashu** [reply]
Hello!

It is working now.
I get the expected information when using "sbctl status" on my computer now.

THANKS!! :D
